lib/cisco_node_utils/radius_global.rb

```diff
@@ -101,7 +101,11 @@ def key_format
     end
 
     def key
-      config_get('radius_global', 'key')
+      str = config_get('radius_global', 'key')
+      return if str.nil?
+      str = str.strip
+      str = "\"#{str}\"" unless str.start_with?('"') && str.end_with?('"')
```

Comment:
Add a helper method in https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cisco_cmn_utils.rb for this and call instead of duplicating.

```ruby
add_quotes(value)
  value = "\"#{value}\"" unless value.start_with?('"') && value.end_with?('"')
  value
end
```